### ISSUE TITLE
Copyright issue

### ISSUE THREAD

**Issue by supriyo-biswas** [MoonchildProductions/PM4Linux-installer#28]
The readme states:

> `*.wrapper` are taken from Debian and used for terminal emulators that do not handle -e arguments properly.

but don't specify a license, thus I looked at:

* http://ftp.us.debian.org/debian/pool/main/x/xfce4-terminal/xfce4-terminal_0.8.3-1_amd64.deb
* http://ftp.us.debian.org/debian/pool/main/g/gnome-terminal/gnome-terminal_3.22.2-1_amd64.deb

The `xfce4-terminal.wrapper` script in particular specifies:

```
# Terminal.wrapper - Debian terminal wrapper script
#
# Copyright (c) 2004-2005 os-cillation
#
```

Since this is not sufficiently clear about the licensing, I also looked at wget http://http.debian.net/debian/pool/main/x/xfce4-terminal/xfce4-terminal_0.8.3-1.debian.tar.xz, and it doesn't specify that this particular file is under an open-source license.

The situation with `gnome-terminal.wrapper` is unclear since there is no `gnome-terminal.wrapper` script in the original source code, there is no licensing info in the wrapper script itself, and wget http://http.debian.net/debian/pool/main/g/gnome-terminal/gnome-terminal_3.22.2-1.debian.tar.xz is also vague about the licensing of this particular file.

So these files should be probably removed? It seems that the installer script only uses the `-e` switch out of the entire functionality, maybe just that part could be reimplemented?

**Comment by wolfbeast** [comment]
If the xfce4 terminal or gnome terminal or wget are published under the GNU GPL, then any file included in those packages are automatically also GNU GPL licensed (it's one of the issues that makes the GPL incompatible with a good number of other Open Source licenses).

Still it would be good to look at this and verify where this file comes from and how it's originally been published before it was taken for this installer.
Question: What is your interest in this that made you bring it up?

**Comment by supriyo-biswas** [comment]
> If the xfce4 terminal or gnome terminal or wget are published under the GNU GPL, then any file included in those packages are automatically also GNU GPL licensed

I hadn't thought of this; thanks for bringing this up. However, it might still be prudent to mention the actual licenses of the wrappers in the LICENSE file.

> What is your interest in this that made you bring it up?

I've been playing around with the idea of building a toy Linux distribution just for the fun of it. As part of this, I wanted to see if there existed a non-policykit based graphical su program. Since this installer implements one, I wanted to see how this was implemented. That's when I noticed the wrapper scripts and the vagueness (at least that's how it appeared to me at the time) of the license.
